packages/flame/examples/vite-tanstack/src/routes/index.tsx

```
@@ -0,0 +1,26 @@
+                <p>
+                    Edit <code>src/routes/index.tsx</code> and save to reload.
+                </p>
+                <a className="text-[#61dafb] hover:underline" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
+                    Learn React
+                </a>
+                <a className="text-[#61dafb] hover:underline" href="https://tanstack.com" target="_blank" rel="noopener noreferrer">
+                    Learn TanStack
+                </a>
+            </header>
+        </div>
+    );
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)